src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,76 @@
+import "./Profile.css";
+import SideBar from "../SideBar/SideBar";
+import { useState } from "react";
+import { searchAlbums } from "../../utils/SpotifyApi";
+
+function Profile() {
+  const [searchInput, setSearchInput] = useState("");
+  const [albums, setAlbums] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchInput(e.target.value);
+  };
+  const handleSearch = (event) => {
+    event.preventDefault();
+
+    const accessToken = localStorage.getItem("spotify_access_token");
+
+    if (accessToken && searchInput.trim() !== "") {
+      searchAlbums(searchInput, accessToken)
+        .then((data) => {
+          setAlbums(data);
+        })
+        .catch((err) => {
+          console.error("Error fetching albums:", err);
+        });
+    }
+  };
+
+  return (
+    <div className="profile">
+      <section className="profile__sidebar">
+        <SideBar />
+      </section>
+      <div className="profile__search">
+        <form className="profile__search_input" onSubmit={handleSearch}>
+          <label htmlFor="myInput">Search:</label>
+          <input
+            type="text"
+            id="myInput"
+            className="profile__search_album"
```

Comment:
`profile__search_album`

and so on


**Modifier** should be used only with its modifiable **basic** class: `block__element block__element_modifier-key_modifier-value`. Elements should get only **modifying styles** from modifier, and all other styles should be got from the basic class.


More info about `BEM` naming can be found here: https://en.bem.info/methodology/quick-start/ 

But if it's not a modifier, then you should not use `_` in it.

`profile__search-album`

**Please, correct it everywhere in the code**